sdk/core/core-paging/review/core-paging.api.md

```diff
@@ -5,7 +5,7 @@
 ```ts
 
 // @public
-export function getPagedAsyncIterator<TElement, TPage = TElement[], TPageSettings = PageSettings>(pagedResult: PagedResult<TPage, TPageSettings>): PagedAsyncIterableIterator<TElement, TPage, TPageSettings>;
+export function getPagedAsyncIterator<TElement, TPage = TElement[], TLink = string, TPageSettings = PageSettings>(pagedResult: PagedResult<TPage, TLink, TPageSettings>): PagedAsyncIterableIterator<TElement, TPage, TPageSettings>;
```

Comment:
I feel like this param should be last, since the type of `TLink` can be inferred from the required parameter `firstPageLink` in `PagedResult`

```suggestion
export function getPagedAsyncIterator<TElement, TPage = TElement[], TPageSettings = PageSettings, TLink = string>(pagedResult: PagedResult<TPage, TLink, TPageSettings>): PagedAsyncIterableIterator<TElement, TPage, TPageSettings>;
```

Reply:
I did the re-ordering but it seems like I still have to specify it still in the tests, otherwise the default will be picked.